src/openforms/js/components/admin/forms/Inputs.js

```diff
@@ -28,6 +29,48 @@ const NumberInput = props => <Input type="number" {...props} />;
 
 const DateInput = props => <Input type="date" {...props} />;
 
+const DateTimeInput = ({name, value, formatDatetime, onChange, ...extraProps}) => {
+  const datetimePickerRef = useRef();
+  const intl = useIntl();
+
+  const wrappedOnChange = (selectedDates, dateStr, instance) => {
+    let formattedDatetime = dateStr;
+    if (formatDatetime) formattedDatetime = formatDatetime(selectedDates[0]);
+
+    const fakeEvent = {
+      target: {name: name, value: formattedDatetime},
+    };
+    onChange(fakeEvent);
+  };
+
+  useEffect(() => {
+    flatpickr(datetimePickerRef.current, {
+      onChange: wrappedOnChange,
+      ...extraProps,
+    });
+  }, []);
+
+  const placeHolder =
+    extraProps.placeholder ||
+    defineMessage({
+      description: 'datetime input widget placeholder',
+      defaultMessage: 'Select date/time...',
+    });
```

Comment:
this means you can't simply pass an already `FormattedMessage` or string to `placeholder`, as it's passed down to `intl.formatMessage`. Maybe it'd be better to use `intl.formatMessage` directly in the default value and then just forward `placeholder`:

```jsx
const placeholder = extraProps.placeholder ?? intl.formatMesage({description: ..., defaultMessage: ...});
...
return (
    <input ref={...} placeholder={placeholder} />
)
```